kimchi/src/verifier.rs

```diff
@@ -803,26 +806,22 @@ where
         return Ok(());
     }
 
-    //~ 1. Ensure that all the proof's verifier index have a URS of the same length. (TODO: do they have to be the same URS though? should we check for that?)
```

Comment:
I don't think this code should be removed : o

also, when you remove this type of comments: `//~` you actually remove parts of the specification (that gets built in https://o1-labs.github.io/proof-systems/specs/kimchi.html#batch-verification-of-proofs (LaTeX there is broken currently))

Reply:
Fair point: if verifier index is passed in the tuple, this code should definitely stay in place. 

(A bit of annoyance with tuple is it's becoming too complex, and the linter doesn't like it, which means a helper datatype is needed.)

Reply:
Rolled this one back.